src/website/stores/Timestamp/TimestampReducers.ts

```diff
@@ -0,0 +1,34 @@
+import BaseReducer from '@/utils/BaseReducer';
+import TimestampActions from './TimestampActions';
+import { TimestampState } from './TimestampTypes';
+import { AnyAction } from 'redux';
+
+export default class TimestampReducer extends BaseReducer {
```

Comment:
Again, we'll need to call this `DataFilterReducer` (and subsequently for the saga and state as well)

Reply:
done